lib/WUI/nhttp/file_info.cpp

```diff
@@ -147,7 +149,22 @@ Step FileInfo::step(std::string_view, bool, uint8_t *output, size_t output_size)
         } else {
             return Step { 0, 0, StatusPage(Status::NotFound, can_keep_alive ? StatusPage::CloseHandling::KeepAlive : StatusPage::CloseHandling::Close, json_errors) };
         }
-        written = write_headers(output, output_size, after_upload ? Status::Created : Status::Ok, ContentType::ApplicationJson, handling);
+
+        static const char *extra_hdrs_no_printing[] = {
+            "Read-Only: false\r\n",
+            "Currently-Printing: false\r\n",
+            nullptr
+        };
+        static const char *extra_hdrs_printing[] = {
+            "Read-Only: false\r\n",
+            "Currently-Printing: true\r\n",
+            nullptr
+        };
```

Comment:
Hmm, this doesn't look great. I think lifetime of the parameter is OK to be just during the call to `write_headers`, so you could use one static string, another buffer on stack to render the header in and have the array also on stack.

Reply:
Ping? Is this a leftover?